examples/server/README.md

```diff
@@ -448,27 +452,48 @@ These words will not be included in the completion, so make sure to add them to
 
 - Note: When using streaming mode (`stream`), only `content` and `stop` will be returned until end of completion.
 
-- `completion_probabilities`: An array of token probabilities for each completion. The array's length is `n_predict`. Each item in the array has the following structure:
+- `completion_probabilities`: An array of token probabilities for each completion. The array's length is `n_predict`. Each item in the array has a nested array `top_logprobs`. It contains at **maximum** `n_probs` elements:
 
 ```json
 {
-  "content": "<the token selected by the model>",
-  "probs": [
+  "content": "<the generated completion text>",
+  ...
+  "completion_probabilities": [
     {
+      "id": <token id>,
       "prob": float,
-      "tok_str": "<most likely token>"
+      "token": "<most likely token>",
+      "bytes": [int, int, ...],
+      "top_logprobs": [
+        {
+          "id": <token id>,
+          "prob": float,
```

Comment:
The surrounding field is called `top_logprobs`, but the inner value is labeled as `prob`. Which is it – a prob or a logprob?

Reply:
update. it should be `logprob` as mentioned in the example (forgot to update it here - thanks for reminding me)